visits-service-new/src/test/java/com/petclinic/visits/visitsservicenew/PresentationLayer/VisitControllerUnitTest.java

```diff
@@ -29,6 +29,21 @@ class VisitControllerUnitTest {
     private final int Pet_Id_OK = visitResponseDTO.getPetId();
     private final int Get_Month = visitResponseDTO.getMonth();
 
+    @Test
+    void getAllVisits(){
+        when(visitService.getAllVisits()).thenReturn(Flux.just(visitResponseDTO, visitResponseDTO));
+
+        webFluxTest.get()
+                .uri("/visits")
+                .accept(MediaType.TEXT_EVENT_STREAM)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.TEXT_EVENT_STREAM + ";charset=UTF-8")
+                .returnResult(VisitResponseDTO.class);
+
+        Mockito.verify(visitService, times(1)).getAllVisits();
+    }
```

Comment:
Controller Unit test is reactive, could check the size of the list that is returned.